### PR TITLE
Clarify @staff role in governance docs

### PR DESCRIPTION
**Pull request by boehs**
## Changes

See commits

## Testing

if (boehs != cool) throw new Error('wrong')

## Docs

Yes, updated GOVERANCE

### CODE REVIEW

**Comment by FredKSchott** [comment]
@boehs thank you for kicking this off! I went ahead and made some edits directly to your PR, I hope that's okay. Give the changes a look, and let me know if you have any questions.

If this looks good to you, I'll go ahead and ping the maintainers team to put this up for a vote before merging. All changes to governance require a 50% approval to pass.

**Comment by boehs** [comment]
I'm sorry to have strayed from your vision! 

- Re !contribute, that did not work for me (could be because I already have the role)
- Not sure why the open collective thing was removed

Thanks for the insight into staff! 
-

File: GOVERNANCE.md
Comment (sarah11918) [inline GOVERNANCE.md:199]: This was the only thing that jumped out at me, that I want to make sure I understand and that it's what is intended.  :)

It would neither surprise nor bother me if Astro staff "earned" their way into core while staff. This line seems to imply that that is not an option. Both Jon and Tony have brought up this point re: their joining staff, wanting to "earn" core and not just immediately receive the governance status because of their employment. But, this clause seems to prevent that from happen, with their status "frozen" as long as they're employed.

So, just checking on this!

File: GOVERNANCE.md
Comment (sarah11918) [inline GOVERNANCE.md:55]: I've never seen this documented anywhere, but I've heard a few people in Discord say that reaching "level 3" automatically confers the contributor role. (And, I know some people have been surprised to see themselves just become a different colour, so maybe that has happened, or maybe core have granted the role after noticing/having someone pointed out to them.

If these two other methods for reaching Contributor role are possible, then maybe they should be documented here, too? (Though, I get that these aren't "nominations" per se.) Just putting it out there.

**Review by RafidMuhymin — APPROVED** [review]

**Review by RafidMuhymin — APPROVED** [review]

**Review by RafidMuhymin — APPROVED** [review]

**Comment by boehs** [comment]
@aFuzzyBear This pull request is a draft, per sarah's comment

File: GOVERNANCE.md
Comment (boehs) [reply GOVERNANCE.md:55]: Yes, contribute once anywhere be a contributor everywhere!

See: https://github.com/withastro/astro/blob/5fde35196467f6e238989af9b8448b39098f0a81/GOVERNANCE.md#examples-of-recognized-contributions

Comment (boehs) [reply GOVERNANCE.md]: resolved in latest commit (ed81328)

**Comment by aFuzzyBear** [comment]
Awesome, sorry if I caused a y confusion, had to make a mad dash away at that time,

**Comment by boehs** [comment]
Why is macos failing, what have I done

File: GOVERNANCE.md
Comment (sarah11918) [inline GOVERNANCE.md:63]: Capital D in Discord.  :P

Comment (boehs) [reply GOVERNANCE.md:63]: aww come on

File: GOVERNANCE.md
Comment (sarah11918) [reply GOVERNANCE.md]: Discord is better than discord

**Review by sarah11918 — COMMENTED** [review]
No new comments on this now! I've seen the changes. I've upper-cased some Ds. Still curious about "joining staff, freezing status" since that doesn't instantly seem consistent to me, so I wanted to make sure to point it out.  :)

File: GOVERNANCE.md
Comment (FredKSchott) [reply GOVERNANCE.md:202]: @tony-sull that's in line with what I was envisioning here! Let me update this line to look more like what you've written to make the distinction between "community" and "staff" more clear.

@sarah11918 yup, that's intentional. What I'm trying to prevent is a #core channel that is filled with staff and ex-staff members. Because staff is "gifted" the same rights as core, this distinction should have no impact on how staff members can function in our community.

The section on `#### Leaving Staff` is meant to address this by saying that anyone leaving staff is still able to jump up to `@core` through the normal nomination channels if they would like.

**Comment by boehs** [comment]
> @boehs can you confirm? That's not what I'm seeing!

See https://mee6.xyz/leaderboard/830184174198718474

Screenshot from 2022-02-14 14-18-24

**Comment by FredKSchott** [comment]
Ohhhh I see what you're saying. I forgot that we were doing that! I now understand why you made that edit to the doc. Thanks for walking me through it :)

File: GOVERNANCE.md
Comment (jonathantneal) [inline GOVERNANCE.md:201]: Could this be limited in scope, say to `@core`?

I’m unaware of any current staff who were not already maintainers before being hired. For instance, I think Tony and I were 'maintainers' beforehand, not 'core', and so I’m unsure what scenario of ex-staff members who were never contributors or maintainers that this is targeting.

However, if I imagine a future where there are whole teams of ex-staff, I am uncomfortable with this for 2 reasons. 1. Would really good interns have to start over? 2. If we became a big ol’ company and I was President and did something _really bad_ like help start a war, I would want a bunch people to be able to quit and keep amazing powers to lead the revolt against us. I may be very biased here having grown up in the US, but I like that at scale this feels like a check and balance.

Disclaimer: I am not running for President of Astro Technology Company, and I have no intention of starting wars, ever.

Comment (FredKSchott) [reply GOVERNANCE.md:201]: haha, I like the visual. All hail our new Astro overlords :)

The idea is to move from **opt-out** ("Hey, I'm leaving staff and am opting out of the core role because I am no longer interested") to **opt-in** ("Hey, I'm leaving staff but would actively like to keep contributing!"). 

The change should have no impact on your **ability** to become a core maintainer post-staff (aka no impact on war scenario, since nominations are a core vote and not at the will of the project steward).

**Review by tony-sull — APPROVED** [review]
Looks good to me!  Thanks for the tweak in wording around staff vs. community roles :+1:

**Review by FredKSchott — APPROVED** [review]

**Review by jonathantneal — APPROVED** [review]

**Review by sarah11918 — APPROVED** [review]

File: GOVERNANCE.md
Comment (natemoo-re) [inline GOVERNANCE.md:43]: Only nit is that I think we just closed this channel, is that correct @jonathantneal?

Comment (jonathantneal) [reply GOVERNANCE.md:43]:
```suggestion
```

**Review by natemoo-re — APPROVED** [review]

File: GOVERNANCE.md
Comment (boehs) [reply GOVERNANCE.md]: thanks, I was thinking about this but forgot to look into

**Comment by FredKSchott** [comment]
Vote passes! Thank you again @boehs for kicking this off

File: GOVERNANCE.md
Comment (jasikpark) [reply GOVERNANCE.md:200]: I appreciate this change; I can imagine that if wanted, any staff that left that still wanted to continue to contribute and increase in role could easily progress quickly anyways...

Comment (jasikpark) [reply GOVERNANCE.md:200]: How is that status tracked? Just by someone belonging to `@/maintainers` and `@/staff` roles in discord and github? And then loses the `@/staff` role when leaving, giving them just `@/maintainers` role + the opportunity to become a core maintainer?

**Review by jasikpark — COMMENTED** [review]
LGTM :)

File: GOVERNANCE.md
Comment (FredKSchott) [reply GOVERNANCE.md:200]: The plan outlined here is: 
- you can be nominated for `maintainer` & `core` before joining staff
- once joining staff, you can no longer be nominated for those roles. (but keep the roles you have)
- once leaving staff, you can once more be nominated for those roles.

**Pull request merged**
